README.adoc

```diff
@@ -4,7 +4,7 @@ GitHub Action to deploy https://kn.dev[Knative Service] using https://github.com
 
 == Pre-requisites
 
-Kubernetes Cluster with Knative, if you dont have an OpenShift cluster get one spinned instantly via https://try.openshift.com[try.openshift.com].
+Kubernetes Cluster with Knative, if you dont have an OpenShift cluster get one using https://try.openshift.com[try.openshift.com].
```

Comment:
Good change but can you also mention the sandbox and link to https://developers.redhat.com/developer-sandbox